src/cmds/init/initialize.ts

```diff
@@ -110,15 +110,16 @@ async function asyncPackageInit() {
   });
 }
 
-async function buildWeb3Wallet(argv) {
+async function buildWeb3Wallet(argv: Argument) {
 
   const { outfileName } = argv;
   const defaultPerms = { alert: {} };
   let { port, dist } = argv;
-  let initialPermissions = defaultPerms;
+  let initialPermissions = defaultPerms as unknown as string;
```

Comment:
This is violating type safety in a bad way, since `defaultPerms` is not a string. Casting `VARIABLE as unnknown as TYPE` is almost as bad as `as any` and should be avoided unless necessary and obviously safe.

Reply:
There are other uses of `defaultPerms as unknown as string` that are questionable in this file.